lte/gateway/deploy/agw_install.sh

```diff
@@ -9,6 +9,7 @@ NEED_REBOOT=0
 WHOAMI=$(whoami)
```

Comment:
Having a pre-built AMI eliminates the user to run these scripts and can be addressed through the devops process. Code exists today in fbcinternal/mantle or https://github.com/magma/magma/pull/4718 and works for 1.3